src/Rbac/DefaultRoleManager/RoleManager.php

```diff
@@ -351,4 +313,20 @@ protected function &loadOrStoreRoles(string $domain, Roles $roles): Roles
 
         return $this->allDomains[$domain];
     }
-}
+
+    /**
+     * @param string $domain
+     *
+     * @return Roles
+     */
+    protected function &checkHasDomainPatternOrHasPattern(string $domain): Roles
```

Comment:
Warning it's not a "check" but more a "get" because you recover all Roles at the end.